client/src/pages/mainlayout/Navlinks.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { Tabs, Tab, makeStyles } from "@material-ui/core";
+import { useLocation, Link } from "react-router-dom";
+
+//CSS styles
+const useStyles = makeStyles((theme) => ({
+  tabContainer: {
+    marginRight: "10rem"
+  },
+  tab: {
+    height: "6rem",
+    padding: "10px",
+    color: "black"
+  },
+  indicator: {
+    display: "flex",
+    justifyContent: "center",
+    height: "5px",
+    top: 0,
+    backgroundColor: "transparent",
+    "& div": {
+      maxWidth: 100,
+      width: "100%",
+      backgroundColor: theme.palette.primary.main
+    }
+  }
+}));
+
+const Navlinks = () => {
+  const { tab, indicator, tabContainer } = useStyles();
+  const location = useLocation();
+
+  const links = [
+    { label: "Campaigns", to: "/campaigns" },
+    { label: "Prospects", to: "/prospects" },
+    { label: "Templates", to: "/templates" },
+    { label: "Reporting", to: "/reporting" }
+  ];
```

Comment:
small suggestion - this may be better to put in a different structure in a constants folder / file, since it's used across multiple files

Reply:
I moved these routes to the constants folder